tests/test.py

```diff
@@ -6,9 +6,11 @@
 live_server_url = "http://127.0.0.1:4000"
 
 routes = [
+    ("blog"),
     ("about"),
-    ("community"),
     ("events"),
+    ("community"),
+    ("meet-our-leaders"),
```

Comment:
```suggestion
("leadership"),
```

Reply:
When I make this change, I encounter a failing test as we don't have a `leadership` route. Should I update the naming convention from `meet-our-leaders.md` to `leadership.md` instead? A 👍 for approval and I'll proceed.